moveit_setup_assistant/templates/moveit_config_pkg_template/launch/ros_controllers.launch

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<launch>
+
+  <!-- Load joint controller configurations from YAML file to parameter server -->
+  <rosparam file="$(find [GENERATED_PACKAGE_NAME])/config/ros_controllers.yaml" command="load"/>
+
+  <!-- Load the controllers -->
+  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
+    output="screen" ns="/[GENERATED_PACKAGE_NAME]" args="spawn joint_state_controller position_trajectory_controller"/>
```

Comment:
@willcbaker Is using absolute namespaces compatible to 4598f6602ded91d91fccabe7a8bcc490a528fc54?

Reply:
I don't believe so, I would advise against absolute namespacing.
Unable to test or review until next week, seems changing this way with generated package name will require pushing down into package name namespace?

Reply:
So you recommend to remove the `ns` argument from here and from the `robot_state_publisher` node? right?

Reply:
Yes, I think so.

Reply:
This `ros_controllers.launch` is intended to be used with `gazebo`, and according to [`ros_control` tutorials](http://gazebosim.org/tutorials/?tut=ros_control#Addthegazebo_ros_controlplugin), this is how `gazebo_ros_control` plugin is defined in the `urdf`
```
<gazebo>
  <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
    <robotNamespace>/MYROBOT</robotNamespace>
  </plugin>
</gazebo>
```
So if we were to launch the controllers with not the `robot_name` as a `namespace` , and leave the plugin element `<robotNamespace>`  blank, how would we be able to use simulate multiple robot for example?
Is there a solution for a situation like that?